README.rst

```diff
@@ -1,6 +1,12 @@
 ========
 Gotthard
 ========
+Gotthard is a command line tool to simplify connecting to a PostgreSQL instance via a bastion_ host.
+This tool is created in the context of zalando-stups_, however it does not depend on it.
```

Comment:
I can weaken it a bit. However, piu is only needed if you specify a reason.

Something like "however it could be used without it".